docker-compose.prod.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+
+services:
+  db:
+    restart: always
+    environment:
+      MYSQL_DATABASE: 'momentum'
+      MYSQL_USER: ${MOM_DATABASE_USER}
+      MYSQL_PASSWORD: ${MOM_DATABASE_PW}
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PW}
```

Comment:
For all these docker files, we set the DB connection info here - why not set this as env vars for the API to fetch? That way it isn't needed in the .env?

Reply:
Not sure what you mean here, can you elaborate more?